README.md

```diff
@@ -22,7 +22,8 @@ docker-machine create --driver amazonec2 kvstore
 ```
 eval $(docker-machine env kvstore)
 export KV_IP=$(docker-machine ip kvstore)
-docker run -d -p 8500:8500 -h consul --restart=always progrium/consul -server -bootstrap
+docker-machine ssh kvstore
```

Comment:
Here is a screenshot from my running those commands. You can see it create the local consul cluster in the screenshot below.

![screen shot 2016-06-06 at 12 03 18 pm](https://cloud.githubusercontent.com/assets/753242/15828830/ea9d5ba2-2bde-11e6-99ef-1d062217bc56.png)